src/app/header/header.component.ts

```diff
@@ -7,14 +7,13 @@ import { TranslateService } from '@ngx-translate/core';
   styleUrls: ['./header.component.scss']
 })
 export class HeaderComponent implements OnInit {
-  id = 0;
-  lang = ["Changer en Français", "Change to English"];
+  lang = "Changer en Français";
```

Comment:
you dont need all of this as i said you can add the translation and itll change dynamically
so in french put the english one and in english put the french one